### DISCUSSION TITLE
Many thanks

### DISCUSSION THREAD

**Discussion by gitothy** [jensrossbach/node-red-contrib-sony-audio#36] [Feedback]
Just wanted to say many thanks for building and maintaining this palette.  I'd been struggling for ages to control individual speaker levels from a dashboard in Home Assistant and your work here was the key. 

![Screenshot 2021-08-31 160049](https://user-images.githubusercontent.com/39539635/131527249-dadbeade-fefe-448c-afef-3c5cac3e2f67.png)

👍

**Comment by jensrossbach** [comment]
Thank you very much for your feedback! I'm happy that the nodes are of such benefit for you.
